grails-app/conf/UrlMappings.groovy

```diff
@@ -70,6 +71,7 @@ class UrlMappings {
 		"/api/v1/users/me"(method: "DELETE", controller: "userApi", action: "delete")
 
 		"/api/v1/users/me/keys"(resources: "keyApi", excludes: ["create", "edit", "update"]) { resourceClass = SecUser }
+		"/api/v1/users/me/keys/$keyId"(method: "PUT", resources: "keyApi", excludes: ["create", "edit", "index", "save"], action: "update") { resourceClass = SecUser }
```

Comment:
`excludes` not needed here. It should only be needed when you use the `resources` argument.